packages/golden_toolkit/test/golden_assertion_test.dart

```diff
@@ -13,9 +13,9 @@ import 'package:golden_toolkit/golden_toolkit.dart';
 
 void main() {
   group('testGoldens validation', () {
-    testWidgets('screenMatchesGolden should require testGoldens',
+    testGoldens('screenMatchesGolden should require testGoldens',
```

Comment:
```suggestion
    testWidgets('screenMatchesGolden should require testGoldens',
```

This was unintentionally committed - here to demonstrate a faulty test (fixed with pumpWidgetBuilder below